docs/examples/metadata_extraction/MetadataExtractionSEC.ipynb

```diff
@@ -150,7 +162,7 @@
    "outputs": [],
    "source": [
     "# Note the uninformative document file name, which may be a common scenario in a production setting\n",
-    "uber_docs = SimpleDirectoryReader(input_files=[\"data/10k-132.pdf\"]).load_data()\n",
+    "uber_docs = SimpleDirectoryReader(input_files=[\"data/uber.pdf\"]).load_data()\n",
```

Comment:
Kind of want to leave out the file name as the point is to show that with poorly cleaned/labeled data, it takes some feature augmentation for the qna to work

Reply:
sounds good, changing back